src/inmanta/protocol/methods_v2.py

```diff
@@ -1285,6 +1284,23 @@ def get_source_code(tid: uuid.UUID, version: int, resource_type: str) -> List[mo
     """
 
 
+@typedmethod(
+    path="/pip/config/<version>",
+    operation="GET",
+    agent_server=True,
+    arg_options=methods.ENV_OPTS,
+    client_types=[ClientType.agent],
+    api_version=2,
+)
+def get_pip_config(tid: uuid.UUID, version: int) -> model.PipConfig:
```

Comment:
Should this return `Optional[PipConfig]` ? If I do this, the agent knows when the model is compiled with an too old version, not it just gets the fallback config?

Reply:
changed